src/main/kotlin/dev/deftu/omnicore/api/chat/OmniChat.kt

```diff
@@ -0,0 +1,131 @@
+package dev.deftu.omnicore.api.chat
+
+import dev.deftu.omnicore.internal.asReadableString
+import dev.deftu.textile.minecraft.MCHoverEvent
+import dev.deftu.textile.minecraft.MCSimpleMutableTextHolder
+import dev.deftu.textile.minecraft.MCSimpleTextHolder
+import dev.deftu.textile.minecraft.MCTextFormat
+import dev.deftu.textile.minecraft.MCTextHolder
+import net.minecraft.server.command.CommandOutput
+import net.minecraft.server.network.ServerPlayerEntity
+
+//#if MC <= 1.8.9
+//$$ import net.minecraft.network.packet.s2c.play.ChatMessageS2CPacket
+//#endif
+
+public object OmniChat {
```

Comment:
What if you dont have an instance of the ServerPlayerEntity? How does this work client side?

Reply:
`OmniClientChat`